### DISCUSSION TITLE
hardhat project in foundry

### DISCUSSION THREAD

**Discussion by sagoto80** [Cyfrin/security-and-auditing-full-course-s23#74] [Q&A] [answered]
how a hardhat project can be tested in foundry

**Comment by johnsonstephan** [comment, marked as answer]
Hi @sagoto80, great question!

> Question: How can a Hardhat project be tested using Foundry?

Answer: You can use the `hardhat-foundry` plugin. 
- Here's the [integration guide](https://book.getfoundry.sh/config/hardhat) on the process from the Foundry documentation
- Here's a "Hardhat in Foundry" [template](https://github.com/foundry-rs/HardhatInFoundry) you can clone to quickly get things moving!

Does this help to answer your question?

**Reply by cromewar** [reply]
You are nailing it @johnsonstephan
